src/main/java/com/wseemann/ecp/request/KeyPressRequest.java

```
@@ -10,6 +14,10 @@ final public class KeyPressRequest extends ECPRequest<Void> {
 	
 	public KeyPressRequest(String url, String key) {
 		super(url);		
+		try {
+		    key = URLEncoder.encode(key, StandardCharsets.UTF_8.name());
+		} catch (UnsupportedEncodingException e) {
```

Can you annotate this constructor as throwing `UnsupportedEncodingException` and let the person implementing the call handle the exception? With this approach the exception is swallowed and it's probably important for the developer to know that we are attempting to encode the key they are passing.